scripts/prci_test_control.py

```diff
@@ -138,5 +140,7 @@ def cmd_rerun(self, args):
         tc(args)
     except ValueError as exc:
         print(exc)
-        parser.print_help()
         exit(2)
+
```

Comment:
PEP 8 requires two empty lines between the end of `main` method and the `if` statement.